### ISSUE TITLE
Makefile missing definition of GIT_TAG

### ISSUE THREAD

**Issue by SeedDMS** [gotify/cli#64]
Add

`GIT_TAG=$(shell git tag)`
to the makefile

**Comment by jmattheis** [comment]
Yeah, this tag was previously supplied by the build system. It certainly could be added as variable there.
